radixdlt/babylon-wallet-android
# Update to ret 1.0.5

## Description
Ret was updated to fall back to non-conforming preview when `try_deposit_batch_or_refund` is used. Previously such manifests where displayed as a normal transaction. Since we update to a newer version there were some additional changes to RET's API surface. 
* All `List<UByte> ` are automatically compiled as `ByteArray` so we don't need to switch between types.
* Some methods in manifest creation have been changed.

## How to test
Try this manifest
```
CALL_METHOD
    Address("component_tdx_2_1crv2sgys7cz9tndj5frynkyt6udf2uwln0yxvwmpq5zvuy2e6zc2wd")
    "free_token"
;
TAKE_FROM_WORKTOP
    Address("resource_tdx_2_1thf262utjrj506kmsadlhy9j2w8q6taenzkev6fup3ssfttrark3ls")
    Decimal("1")
    Bucket("bucket");

CALL_METHOD
    Address("<YOU ACCOUNT ADDRESS>")
    "deposit"
    Bucket("bucket");

CALL_METHOD
    Address("<YOU ACCOUNT ADDRESS>")
    "try_deposit_batch_or_refund"
    Expression("ENTIRE_WORKTOP")
    None
;
```
In the previous version, this manifest was displayed as a normal conforming transaction. Now we should display the raw manifest since RET returns non-conforming type.

Also don't forget to sync gradle and that you gpr keys are not expired so you can download the new library.

Also test faucet in one of your accounts.

## PR submission checklist
- [X] I have tested faucet and it works
